index.js

```diff
@@ -0,0 +1,4 @@
+const prentElement = document.querySelector("ul");
```

Comment:
Q
why did you use querySelector instead of getElementById

Reply:
Thanks for the review!
Now that I know I can add the ID attribute, I will add it and fix the problem.